src/linter/block_linter.go

```diff
@@ -1145,10 +1145,6 @@ func (b *blockLinter) checkFormatString(e *ir.FunctionCallExpr, arg *ir.Argument
 	}
 }
 
-func (b *blockLinter) isArrayType(typ types.Map) bool {
```

Comment:
If we refactor the types in the future, I don't really want to 
have such functions somewhere outside the `types` package, 
since they then need to be looked for, so I moved them to 
the `types.Map` method.